catalog-info.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: open-edx-backstage
+  description: An example of a Backstage application.
+  # Example for optional annotations
+  # annotations:
+  #   github.com/project-slug: backstage/backstage
+  #   backstage.io/techdocs-ref: dir:.
+spec:
+  type: website
+  owner: john@example.com
```

Comment:
This file needs to be updated with a valid owner, I'm happy to volunteer or work with you as co-maintainer.

Reply:
Sounds good, feels like we should have a team for that.